docker_exposure.sh

```diff
@@ -2,11 +2,13 @@
 source run_container.sh
 
 if [ $# -lt 2 ]; then
-    echo "usage: $0 <data_dir> <output_dir> <extra-args>"
+    echo "usage: $0 <input_config_yml_file_path> <output_dir> <extra-args>"
```

Comment:
We  no longer use extra-args, right? Since they come through the yaml?
```suggestion
    echo "usage: $0 <input_config_yml_file_path> <output_dir>"
```

Reply:
updated.